client/my-sites/customer-home/cards/tasks/site-setup-list/current-task-item.jsx

```diff
@@ -14,11 +14,12 @@ const CurrentTaskItem = ( { currentTask, skipTask, startTask, useAccordionLayout
 	return (
 		<div className="site-setup-list__task task">
 			<div className="site-setup-list__task-text task__text">
-				{ currentTask.isCompleted ? (
+				{ currentTask.isCompleted && ! currentTask.hideLabel && (
```

Comment:
I added another prop here so that we preserve the true `isCompleted` value for `recordTracksEvent`